src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -79,7 +102,7 @@ const NavHeader = (props) => {
           ? dimen.size.TOPIC_FEED_HEADER_HEIGHT
           : dimen.size.TOPIC_FEED_HEADER_HEIGHT + normalize(4);
     }
-    return Math.round((containerHeight - domainHeight) / 2);
+    return Math.round((containerHeight - domainHeight) / 2) + searchHeight;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The calculation for return value has been updated to include `searchHeight`. Ensure this change doesn't break the layout or positioning of elements.
```diff
-    return Math.round((containerHeight - domainHeight) / 2);
+    return Math.round((containerHeight - domainHeight) / 2) + searchHeight;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->